Documentation/Books/AQL/Examples/RemoveVertex.md

```diff
@@ -0,0 +1,54 @@
+Remove Vertex
+=============
+
+Deleting vertices with associated edges is currently not handled via AQL while the [graph management interface](../../Manual/Graphs/GeneralGraphs/Management.html#remove-a-vertex) and the [REST API for the graph module](../../HTTP/Gharial/Vertices.html#remove-a-vertex) offer a vertex deletion functionality.
+However, as shown in this example based on the [knows_graph](../../Manual/Graphs/index.html#the-knowsgraph), a query for this use case can be created.
+
+![Example Graph](../../Manual/Graphs/knows_graph.png)
+
+When deleting vertex **eve** from the graph, we also want the edges `eve -> alice` and `eve -> bob` to be removed.
+The involved graph and edge collections have to be known. In this case it is the graph **knows_graph** and the edge collection **knows**.
+
+This query is deleting **eve** with its adjacent edges:
+
+    @startDocuBlockInline GRAPHTRAV_removeVertex1
+    @EXAMPLE_AQL{GRAPHTRAV_removeVertex1}
+    @DATASET{knows_graph}
+LET keys = (FOR v, e IN 1..1 ANY 'persons/eve' GRAPH 'knows_graph' RETURN e._key)
+LET r = (FOR key IN keys REMOVE key IN knows) 
+REMOVE 'eve' IN persons
+    @END_EXAMPLE_AQL
+    @endDocuBlock GRAPHTRAV_removeVertex1
+
+At first a graph traversal of depth 1 is used to get the `_key` of **eve's** adjacent edges.
```

Comment:
make a bullet point list ala 
This query executed several actions: 
 - use a graph traversal of depth 1 to ... 
 - ...